api/models/Community.js

```diff
@@ -90,10 +90,26 @@ module.exports = bookshelf.Model.extend({
       return newPost.save({created_at: time, updated_at: time}, {transacting: trx})
       .then(() => Promise.all(_.flatten([
         self.posts().attach(newPost, {transacting: trx}),
+        Tag.updateForPost(newPost, newPost.get('type'), trx),
```

Comment:
not totally sure it's the root cause, but you have a few finds and loads downstream that don't use the transaction